pytest_ordering/version.py

```diff
@@ -0,0 +1 @@
+__version__ = '1.0'
```

Comment:
Can you change the version back? I bump it in a release branch when I go to release.